api/api.go

```diff
@@ -56,9 +88,62 @@ func GetTechZoneToken(c *gin.Context) {
 		auth.ErrorGettingToken()
 		return
 	}
-	// Serve the HTML page 
+	// Serve the HTML page
 	c.HTML(http.StatusOK, "index.html", requestJson)
 	// Write the token to the config file
-	auth.SaveTokenToConfig(requestJson.Token)
+	_ = auth.SaveTokenToConfig(requestJson.Token)
 }
 
+func CliToRESTHandler(router *gin.Engine) {
+	for _, cobraCmd := range RootCmd.Commands() {
+		// What we want is `itz solution list --list-all` becomes
+		// <url>/api/itz/solution/list&list-all=true
+		baseCmdName := cobraCmd.Name()
+		if baseCmdName == "workspace" || baseCmdName == "completion" || baseCmdName == "api" {
```

Comment:
You may want to turn this into an array or list and check to see if the path is contained in the list--only because then you could have the list as a constant at the top and make it easy to update. Such "ignoredPaths", etc. This for now is probably faster, execution-wise.